documentation/modules/proc-create-address-console.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// master.adoc
+
+[id='create-address-console-{context}']
+= Creating addresses using the {ProductName} Console
+
+You can create new addresses using the {ProductName} Console. The type of addresses that you can create are determined by the type of address space.
+
+ifdef::Supported[]
+For more information see the xref:ref-supported-features-table-messaging[].
+endif::Supported[]
+
+.Prerequisites
+* You must have created an address space. For more information see xref:create-address-space-cli-okd[]
+
+.Procedure
+
+. Log in to the {ProductName} Console.
+
+. Click *New*. The Create new address window opens.
+
+. Type a name and select the address type. If selecting *subscription*, select the topic name from the *Topic* list to which you want to create a subscription.
```

Comment:
'to which you want to create a subscription.' is a bit clunky, consider rewording